CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+How to contribute
+==================
+Thank you for wanting to contribute to Spine. The following links will help you get started:
+ * [Wiki home][wiki-home] — the home of the framework developer's documentation.
+ * [Getting started with Spine in Java][quick-start] — this guide will walk you through
+   a minimal client-server “Hello World!” application in Java. 
+ * [Introduction][docs-intro] — this section of the Spine Documentation will help you understand
+   the foundation of the framework.
+   
+Pull requests
+-------------
+A work on an improvement starts from creating an issue which describes a bug or a feature you
+intend to fix. The issue will be used for communications on the proposed improvements. If code
+changes are going to be introduced, the issue would also have a link to corresponding Pull Request.
```

Comment:
I'd just drop the `intend to fix` part, cause we do not `fix` features and if we add `implement` it'd be just more cluttered.

```suggestion
The work on an improvement starts with creating an issue that describes a bug or a feature. The issue will be used for communications on the proposed improvements. 
If code changes are going to be introduced, the issue should also have a link to the corresponding Pull Request.
```